### PR TITLE
Code Formatting

### PR DESCRIPTION
**Pull request by Chii-Onan**
Just improving the readability by formatting 31 files!

### CODE REVIEW

**Review by VelvetThePanda — CHANGES_REQUESTED** [review]
While some of this is actually formatting, I'm not sure I agree with a lot of the in/outdenting of brackets/braces being added, as it actually does the *opposite* of improving readability. 

Furthermore, there are a lot of questionable changes to some logic, but the git diff is hard to read, so I won't comment on anything specifically. Regardless, this PR scopes out more than the title or description imply.

File: Backend/Remora.Discord.API/API/Gateway/Commands/ConnectionProperties.cs [hunk 1]
```diff
@@ -54,12 +54,12 @@ private static string GetCurrentOSPlatformName()
             return "linux";
         }
 
-        #if NET5_0_OR_GREATER
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/API/Gateway/Commands/ConnectionProperties.cs:57]: This indent is intentional; revert.

File: Backend/Remora.Discord.API/Json/Converters/Internal/DiscordPermissionSetConverter.cs [hunk 2]
```diff
@@ -45,24 +45,24 @@ JsonSerializerOptions options
         switch (reader.TokenType)
         {
             case JsonTokenType.String:
-            {
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/Internal/DiscordPermissionSetConverter.cs:48]: Case braces shall be included and shall not be indented beyond the level of the case itself; revert.

File: Backend/Remora.Discord.API/Json/Converters/Internal/InvalidSessionConverter.cs [hunk 3]
```diff
@@ -38,13 +38,13 @@ public override IInvalidSession Read(ref Utf8JsonReader reader, Type typeToConve
         {
             case JsonTokenType.True:
             case JsonTokenType.False:
-            {
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/Internal/InvalidSessionConverter.cs:41]: See previous comment; revert.

File: Backend/Remora.Discord.API/Json/Converters/Internal/MessageComponentConverter.cs [hunk 4]
```diff
@@ -82,29 +82,29 @@ public override void Write(Utf8JsonWriter writer, IMessageComponent value, JsonS
         switch (value)
         {
             case IActionRowComponent actionRow:
-            {
-                JsonSerializer.Serialize(writer, actionRow, options);
-                break;
-            }
+                {
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/Internal/MessageComponentConverter.cs:85]: See previous comment; revert.

File: Backend/Remora.Discord.API/Json/Converters/Internal/PartialMessageComponentConverter.cs [hunk 5]
```diff
@@ -82,29 +82,29 @@ public override void Write(Utf8JsonWriter writer, IPartialMessageComponent value
         switch (value)
         {
             case IPartialActionRowComponent actionRow:
-            {
-                JsonSerializer.Serialize(writer, actionRow, options);
-                break;
-            }
+                {
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/Internal/PartialMessageComponentConverter.cs:85]: See previous comment; revert.

File: Backend/Remora.Discord.API/Json/Converters/Internal/PayloadConverter.cs [hunk 6]
```diff
@@ -185,23 +185,23 @@ public override void Write(Utf8JsonWriter writer, IPayload? value, JsonSerialize
         switch (payloadData)
         {
             case Reconnect or HeartbeatAcknowledge:
-            {
-                writer.WriteNullValue();
-                break;
-            }
+                {
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/Internal/PayloadConverter.cs:188]: See previous comment; revert.

File: Backend/Remora.Discord.API/Json/Converters/StringEnumListConverter.cs [hunk 7]
```diff
@@ -70,21 +70,21 @@ public override IReadOnlyList<TEnum> Read(ref Utf8JsonReader reader, Type typeTo
             switch (reader.TokenType)
             {
                 case JsonTokenType.String:
-                {
-                    if (_enumValuesByJsonValue.TryGetValue(reader.GetString()!, out var enumValue))
                     {
-                        enumValues.Add(enumValue);
+                        if (_enumValuesByJsonValue.TryGetValue(reader.GetString()!, out var enumValue))
```
Comment (Nihlus) [inline Backend/Remora.Discord.API/Json/Converters/StringEnumListConverter.cs:74]: See previous comment; revert.

File: Backend/Remora.Discord.Gateway/DiscordGatewayClient.cs [hunk 8]
```diff
@@ -323,110 +323,110 @@ private bool ShouldReconnect
         switch (iterationResult.Error)
         {
             case GatewayDiscordError gde:
-            {
```
Comment (Nihlus) [inline Backend/Remora.Discord.Gateway/DiscordGatewayClient.cs:326]: See previous comments; revert entire file.

File: Backend/Remora.Discord.Gateway/Extensions/ServiceCollectionExtensions.cs [hunk 9]
```diff
@@ -93,8 +93,8 @@ public static IServiceCollection AddDiscordGateway
     /// <returns>The service collection, with the responder added.</returns>
     public static IServiceCollection AddResponder
         <
-            [MeansImplicitUse(ImplicitUseKindFlags.InstantiatedNoFixedConstructorSignature)]
-            TResponder
+    [MeansImplicitUse(ImplicitUseKindFlags.InstantiatedNoFixedConstructorSignature)]
```
Comment (Nihlus) [inline Backend/Remora.Discord.Gateway/Extensions/ServiceCollectionExtensions.cs:96]: Indents shall follow the allman style; revert.

File: Backend/Remora.Discord.Gateway/Services/ResponderDispatchService.cs [hunk 10]
```diff
@@ -365,31 +365,31 @@ private void FinalizeResponderDispatch(IEnumerable<Result> responderResults)
                 switch (responderResult.Error)
                 {
                     case ExceptionError exe:
-                    {
-                        if (exe.Exception is OperationCanceledException)
                         {
-                            continue;
+                            if (exe.Exception is OperationCanceledException)
```
Comment (Nihlus) [inline Backend/Remora.Discord.Gateway/Services/ResponderDispatchService.cs:369]: See previous comment; revert.

File: Backend/Remora.Discord.Gateway/Transport/WebSocketPayloadTransportService.cs [hunk 11]
```diff
@@ -130,18 +130,18 @@ public async Task<Result> ConnectAsync(Uri endpoint, CancellationToken ct = defa
             {
                 case WebSocketState.Open:
                 case WebSocketState.Connecting:
-                {
-                    break;
-                }
+                    {
```
Comment (Nihlus) [inline Backend/Remora.Discord.Gateway/Transport/WebSocketPayloadTransportService.cs:133]: See previous comment; revert.

File: Backend/Remora.Discord.Rest/Extensions/RestRequestBuilderExtensions.cs [hunk 12]
```diff
@@ -84,11 +84,11 @@ public static RestRequestBuilder SkipAuthorization
         this RestRequestBuilder builder
     )
     {
-        #if NET5_0_OR_GREATER
+#if NET5_0_OR_GREATER
```
Comment (Nihlus) [inline Backend/Remora.Discord.Rest/Extensions/RestRequestBuilderExtensions.cs:87]: See previous comment; revert.

File: Backend/Remora.Discord.Rest/Handlers/TokenAuthorizationHandler.cs [hunk 13]
```diff
@@ -57,11 +57,11 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             throw new InvalidOperationException("The authentication token has to contain something.");
         }
 
-        #if NET5_0_OR_GREATER
+#if NET5_0_OR_GREATER
```
Comment (Nihlus) [inline Backend/Remora.Discord.Rest/Handlers/TokenAuthorizationHandler.cs:60]: See previous comment; revert.

File: Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs [hunk 14]
```diff
@@ -260,88 +260,88 @@ private static Result
         switch (node)
         {
             case GroupNode groupNode:
-            {
```
Comment (Nihlus) [inline Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs:263]: See previous comment; revert entire file.

File: Remora.Discord.Commands/Extensions/ServiceCollectionExtensions.cs [hunk 15]
```diff
@@ -272,8 +272,8 @@ public static IServiceCollection AddInteractionResponder
     /// <returns>The collection, with the event.</returns>
     public static IServiceCollection AddPreExecutionEvent
         <
-            [MeansImplicitUse(ImplicitUseKindFlags.InstantiatedNoFixedConstructorSignature)]
-            TEvent
+    [MeansImplicitUse(ImplicitUseKindFlags.InstantiatedNoFixedConstructorSignature)]
```
Comment (Nihlus) [inline Remora.Discord.Commands/Extensions/ServiceCollectionExtensions.cs:275]: See previous comment; revert.

File: Remora.Discord.Commands/Feedback/Services/FeedbackService.cs [hunk 16]
```diff
@@ -632,27 +632,27 @@ public Task<Result<IMessage>> SendAsync
         switch (_contextInjection.Context)
         {
             case MessageContext messageContext:
-            {
-                return await SendAsync(messageContext.ChannelID, content, embeds, options, ct);
-            }
+                {
```
Comment (Nihlus) [inline Remora.Discord.Commands/Feedback/Services/FeedbackService.cs:635]: See previous comment; revert.

File: Remora.Discord.Commands/Services/SlashService.cs [hunk 17]
```diff
@@ -58,7 +58,8 @@ public IReadOnlyDictionary
     <
         (Optional<Snowflake> GuildID, Snowflake CommandID),
         OneOf<IReadOnlyDictionary<string, CommandNode>, CommandNode>
-    > CommandMap { get; private set; }
+    > CommandMap
```
Comment (Nihlus) [inline Remora.Discord.Commands/Services/SlashService.cs:61]: See previous comment; revert.

File: Remora.Discord.Hosting/Services/DiscordService.cs [hunk 18]
```diff
@@ -76,34 +76,34 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             switch (runResult.Error)
             {
                 case ExceptionError exe:
-                {
```
Comment (Nihlus) [inline Remora.Discord.Hosting/Services/DiscordService.cs:79]: See previous comment; revert.

File: Remora.Discord.Interactivity/Responders/InteractivityResponder.cs [hunk 19]
```diff
@@ -218,37 +218,37 @@ IEnumerable<IPartialMessageComponent> components
             switch (component)
             {
                 case IPartialTextInputComponent textInput:
-                {
```
Comment (Nihlus) [inline Remora.Discord.Interactivity/Responders/InteractivityResponder.cs:221]: See previous comment; revert.

File: Samples/Autocomplete/Program.cs [hunk 20]
```diff
@@ -42,7 +42,7 @@ public static async Task Main(string[] args)
 
         Snowflake? debugServer = null;
 
-        #if DEBUG
+#if DEBUG
```
Comment (Nihlus) [inline Samples/Autocomplete/Program.cs:45]: See previous comment; revert.

File: Samples/Interactivity/Program.cs [hunk 21]
```diff
@@ -47,7 +47,7 @@ public static async Task Main(string[] args)
 
         Snowflake? debugServer = null;
 
-        #if DEBUG
+#if DEBUG
```
Comment (Nihlus) [inline Samples/Interactivity/Program.cs:50]: See previous comment; revert.

File: Samples/SlashCommands/Program.cs [hunk 22]
```diff
@@ -44,7 +44,7 @@ public static async Task Main(string[] args)
 
         Snowflake? debugServer = null;
 
-        #if DEBUG
+#if DEBUG
```
Comment (Nihlus) [inline Samples/SlashCommands/Program.cs:47]: See previous comment; revert.

File: Tools/Remora.Discord.SensitiveDataScrubber/Program.cs [hunk 23]
```diff
@@ -181,99 +181,99 @@ JsonNode node
         switch (node)
         {
             case JsonObject jsonObject:
-            {
```
Comment (Nihlus) [inline Tools/Remora.Discord.SensitiveDataScrubber/Program.cs:184]: See previous comment; revert.

**Review by Nihlus — CHANGES_REQUESTED** [review]
When submitting pull requests that change formatting, it's important that you understand and follow the conventions used by the rest of the codebase. I've marked and explained the impermissible format changes which need to be reverted.

**Pull request closed without merging**